scripts/docker/CC-Devel/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:16.04
+
+RUN \
+    apt-get update \
+    && apt-get install --yes \
+    openjdk-8-jdk-headless \
+    libboost-filesystem1.58.0 \
+    libboost-log1.58.0 \
+    libboost-program-options1.58.0 \
+    libboost-thread1.58.0 \
+    pkg-config \
+    ant \
```

Comment:
There are some packages here that are not in the manual-install dependency list. What are the reason for those? `ant`, `xz-utils`, etc.

Reply:
That time when these statements was written there was no official downloadable of libthrift 0.11.0 java. So I had to made jar files during image build. The pkgconf will be necessary when thrift jar file dependencies will be removed from CodeCompass repo. The ant will not necessary if the install_thrift.sh script download the necessary jars instead of compile them.